src/components/Person/Person.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+class Person extends React.Component {
+
+  render() {
+    const { person: { id, name, sex, age, born, died, century, father, mother, children },
+      onClick, selectedId } = this.props;
+
+    return (
+      <tr
+        active={selectedId === id}
```

Comment:
`active` хоче не boolean значення
```suggestion
        active={selectedId === id ? 'true' : 'false'}
```